smart-contracts/contracts/Unlock.sol

```diff
@@ -452,24 +452,13 @@ contract Unlock is UnlockInitializable, UnlockOwnable {
               udtPrice;
 
           // or tokensToDistribute is capped by network GDP growth
-          uint maxTokens = 0;
-          if (chainId > 1) {
-            // non mainnet: we distribute tokens using asymptotic curve between 0 and 0.5
-            // maxTokens = IMintableERC20(udt).balanceOf(address(this)).mul((valueInETH / grossNetworkProduct) / (2 + 2 * valueInETH / grossNetworkProduct));
-            maxTokens =
-              (IMintableERC20(udt).balanceOf(
-                address(this)
-              ) * valueInETH) /
-              (2 + (2 * valueInETH) / grossNetworkProduct) /
-              grossNetworkProduct;
-          } else {
-            // Mainnet: we mint new token using log curve
-            maxTokens =
-              (IMintableERC20(udt).totalSupply() *
-                valueInETH) /
-              2 /
-              grossNetworkProduct;
-          }
+          // we distribute tokens using asymptotic curve between 0 and 0.5
+          // maxTokens = IMintableERC20(udt).balanceOf(address(this)).mul((valueInETH / grossNetworkProduct) / (2 + 2 * valueInETH / grossNetworkProduct));
+          uint maxTokens = (IMintableERC20(udt).balanceOf(
+              address(this)
+            ) * valueInETH) /
+            (2 + (2 * valueInETH) / grossNetworkProduct) /
+            grossNetworkProduct;
```

Comment:
I wonder if there is a way to simplify this equation?

Reply:
that may mean change how UDT distribution works . Should write a proposal about tokenomics first as it is a change in the protocol itself?

Reply:
No no, I want to keep the same formula but have the code be more legible!

Reply:
Ok. A simple thing is to reuse the `balance` variable below. We are hitting "stack too deep" so can not really add more named vars :/